wordpress-mobile/WordPress-Android
# Fix title TalkBack announcements in Login and Site creation sections

Partially fixes #7370

Fixes screen titles in the Login, sign up, site creation and share to WordPress sections.

To test:

turn on TalkBack
1. Enter each screen and make sure, that it's title is announced (the title should be announced also when you get back to the screen - either from a nested activity or from another app/system ui)
